src/ru.ts

```diff
@@ -0,0 +1,2508 @@
+/**
+ * Copyright Clave - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+
+export const ru = {
+    locale: "ru",
+    general: {
+        identifiers: {
+            self: "Я",
+        },
+    },
+    error_codes: {
```

Comment:
These errors are also displayed on UI at some points. Could you please add those too?

Reply:
on it